pkg/dsdk/remote_provider.go

```diff
@@ -229,3 +229,27 @@ func (e *RemoteProvider) Reload(ro *RemoteProviderReloadRequest) (*RemoteProvide
 	RegisterRemoteProviderEndpoints(resp)
 	return resp, nil, nil
 }
+
+type RemoteProviderOperationsUpdateRequest struct {
+	Ctxt        context.Context `json:"-"`
+	OperationId string          `json:"-"`
+	Action      string          `json:"action"` //available options are 'clear' and 'abort'
+}
+
+func (e *RemoteProvider) CreateOperation(ao *RemoteProviderOperationsUpdateRequest) (*RemoteProvider, *ApiErrorResponse, error) {
```

Comment:
Would  `UpdateOperation` be more accurate?

Reply:
This was changed (including the request object) to `SetOperation` to keep in line with similar function calls in the API